src/API tests/RESTful Booker API.postman_collection.json

```diff
@@ -0,0 +1,466 @@
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "27479978",
+		"_collection_link": "https://crimson-star-683354.postman.co/workspace/QAMP2023~1fde8c71-a39d-4f6c-9415-da832a962dca/collection/27479978-4d963ec3-9aea-49cd-b590-b1bde5eb4c4b?action=share&creator=27479978&source=collection_link"
+	},
+	"item": [
+		{
+			"name": "Smoke test for RESTful Booker API",
+			"item": [
+				{
+					"name": "API endpoint health",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 201\", function () {\r",
+									"    pm.response.to.have.status(201);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "https://restful-booker.herokuapp.com/ping",
+							"protocol": "https",
+							"host": [
+								"restful-booker",
+								"herokuapp",
+								"com"
+							],
+							"path": [
+								"ping"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Creating a new auth token",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"\r",
+									"pm.test(\"Response contains token\", function () {\r",
+									"    pm.expect(pm.response.json()).to.have.property(\"token\").and.to.not.be.empty;\r",
+									"    \r",
+									"    var token = pm.response.token;\r",
+									"    pm.collectionVariables.set(\"token\", token);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"username\" : \"admin\",\r\n    \"password\" : \"password123\"\r\n}"
```

Comment:
We should not provide any personal information for valid login even if it is dummy data we should still follow the convention.

Reply:
Yes, I realized and updated it in version for PlaceLab but forget to do it on this. Noted and thanks for heads up.